### PR TITLE
Update rule descriptions

### PR DESCRIPTION
**Pull request by DeadlyMC**
- Added several missing full stops
- Potentially better explain certain rules

### CODE REVIEW

File: src/main/java/carpetextra/CarpetExtraSettings.java [hunk 1]
```diff
@@ -100,77 +99,77 @@ public Integer validate(ServerCommandSource source, ParsedRule<Integer> currentR
...
     @Rule(
-            desc = "Client can provide alternative block placement",
+            desc = "Client can provide alternative block placement.",
             category = {EXTRA, SURVIVAL}
     )
     public static boolean accurateBlockPlacement = false;
 
     @Rule(
-            desc = "Dispensers can toggle with a stick things like redstone, noteblocks, comparator, daylight detector, repeaters",
+            desc = "Dispensers containing a stick can toggle things like buttons, redstone, noteblocks, comparators, daylight detectors and repeaters.",
```
Comment (gnembon) [inline src/main/java/carpetextra/CarpetExtraSettings.java:129]: Suggest splitting this one - won't fit in one line in chat I think

File: src/main/java/carpetextra/CarpetExtraSettings.java [hunk 2]
```diff
@@ -100,77 +99,77 @@ public Integer validate(ServerCommandSource source, ParsedRule<Integer> currentR
...
+            desc = "Re-adds 1.8 double retraction to pistons.",
             category = {EXTRA, EXPERIMENTAL},
-            extra = { "Gives pistons the ability to double retract without side effects" }
+            extra = { "Gives pistons the ability to double retract without side effects." }
     )
     public static boolean doubleRetraction = false;
     
     @Rule(
-            desc = "Fixes block states in F3 debug mode not updating for some blocks. May cause increased network traffic",
+            desc = "Fixes block states in F3 debug mode not updating for some blocks. May cause increased network traffic.",
```
Comment (gnembon) [inline src/main/java/carpetextra/CarpetExtraSettings.java:158]: You might split this one as well, "May.." make extra out of it.

File: src/main/java/carpetextra/CarpetExtraSettings.java [hunk 3]
```diff
@@ -179,57 +178,59 @@ public Integer validate(ServerCommandSource source, ParsedRule<Integer> currentR
...
+                    "All the loaded chunks can be seen using `/forceload query`"
+            },
             category = {EXTRA, FEATURE, EXPERIMENTAL}
     )
     public static boolean flowerPotChunkLoading = false;
     
     @Rule(
             desc = "Won't let mobs glitch into blocks when reloaded.",
-            extra = "Can cause slight differences in mobs behaviour. Fixes MC-2025",
+            extra = "Can cause slight differences in mobs behaviour. Fixes [MC-2025](https://bugs.mojang.com/browse/MC-2025).",
```
Comment (gnembon) [inline src/main/java/carpetextra/CarpetExtraSettings.java:204]: Split this one as well. "Fixes ... link" as an extra line

File: src/main/java/carpetextra/CarpetExtraSettings.java [hunk 4]
```diff
@@ -179,57 +178,59 @@ public Integer validate(ServerCommandSource source, ParsedRule<Integer> currentR
...
+            desc = "Dispensers with empty buckets can milk cows and get stew from mooshrooms with bowls.",
             category = {EXPERIMENTAL, EXTRA, FEATURE, DISPENSER}
     )
     public static boolean dispensersMilkCows = false;
     
     @Rule(
-            desc = "Quick pulses won't get lost in repeater setups",
-            extra = "Probably brings back pre 1.8 behaviour. Fixes MC-54711",
+            desc = "Quick pulses won't get lost in repeater setups.",
+            extra = "Probably brings back pre 1.8 behaviour. Fixes [MC-54711](https://bugs.mojang.com/browse/MC-54711).",
```
Comment (gnembon) [inline src/main/java/carpetextra/CarpetExtraSettings.java:217]: same split

File: src/main/java/carpetextra/CarpetExtraSettings.java [hunk 5]
```diff
@@ -247,61 +248,61 @@ public Integer validate(ServerCommandSource source, ParsedRule<Integer> currentR
...
 
     @Rule(
-            desc = "Dispensers with shears can carve pumpkins",
+            desc = "Dispensers containing shears can carve pumpkins.",
             category = {EXTRA, FEATURE, DISPENSER}
     )
     public static boolean dispensersCarvePumpkins = false;
     
-    @Rule(desc = "Blaze powder fertilizes netherwart via dispenser or player right click action", category = {EXTRA, FEATURE, SURVIVAL})
+    @Rule(desc = "Blaze powder fertilizes netherwart via dispenser or player right click actions.", category = {EXTRA, FEATURE, SURVIVAL})
```
Comment (gnembon) [inline src/main/java/carpetextra/CarpetExtraSettings.java:305]: I would split that too. "... via" into extra

**Review by gnembon — APPROVED** [review]

File: src/main/java/carpetextra/CarpetExtraSettings.java [hunk 6]
```diff
@@ -100,77 +99,78 @@ public Integer validate(ServerCommandSource source, ParsedRule<Integer> currentR
...
-            desc = "Client can provide alternative block placement",
+            desc = "Client can provide alternative block placement.",
             category = {EXTRA, SURVIVAL}
     )
     public static boolean accurateBlockPlacement = false;
 
     @Rule(
-            desc = "Dispensers can toggle with a stick things like redstone, noteblocks, comparator, daylight detector, repeaters",
+            desc = "Dispensers containing a stick can toggle things.",
+            extra = {"Works with buttons, redstone, noteblocks, comparators, repeaters", "and daylight detectors."},
```
Comment (Fallen-Breath) [inline src/main/java/carpetextra/CarpetExtraSettings.java:130]: dispensersToggleThings not only works with these listed blocks. maybe adding a `etc.` or something will be better

Comment (DeadlyMC) [reply src/main/java/carpetextra/CarpetExtraSettings.java:130]: Oh, i thought these where the only blocks...i will add the etc directly as a commit, im sure gnembon wont mind :p

Comment (DeadlyMC) [reply src/main/java/carpetextra/CarpetExtraSettings.java:130]: Resolved in https://github.com/gnembon/carpet-extra/commit/f87c015e0b6165ef9e5e6a024bc32c7f5d255ee8

**Pull request merged**